### ISSUE TITLE
bug(MatMenu): With NVDA enabled, keyboard focus doesn't show up

### ISSUE THREAD

**Issue by calebegg** [angular/components#21947]
Repros on https://material.angular.io/components/menu/overview

Everything works fine when NVDA is disabled

Environment: NVDA version - 2019.2.1 - Firefox Version 83.0 - Windows 10 Enterprise (64-bit)

![image](https://user-images.githubusercontent.com/782920/108384296-9f096600-71d8-11eb-9d31-5a7c9befb485.png)

**Comment by jelbourn** [comment]
@calebegg I just tried this with NVDA 2020.3 and Firefox 85 and I don't see any difference between the focus treatment with and without NVDA enabled. Could have been an issue with an older browser + screen-reader combination?

**Comment by calebegg** [comment]
Still repros consistently for me with those versions (2020.3 and 85.0.2). My specific repro steps

1) start nvda
2) start firefox (order might matter?)
3) go to test page
4) tab to menu trigger
5) press enter

I have video if it would help? Github won't let me upload it here.

**Comment by crisbeto** [comment]
I just tried it with NVDA + Firefox and I wasn't able to reproduce it either.

**Comment by calebegg** [comment]
Could I trouble one of you for a video call to see if we can figure out what we're doing differently then?  Our a11y tester and I are having trouble with this and it's going to block a launch if I can't figure it out.

**Comment by crisbeto** [comment]
Sure, are you on the Angular Slack?

**Comment by calebegg** [comment]
Yep. I'm calebegg on there too.
